src/main/java/com/team1816/season/subsystems/Turret.java

```diff
@@ -411,23 +368,28 @@ private void trackAbsolute() {
         }
     }
 
-    private void positionControl(int rawPos) {
+    private void positionControl(int pos) {
         if (outputsChanged) {
-            if (rawPos > FWD_WRAPAROUND_POINT) {
-                rawPos -= TURRET_PPR;
-            } else if (rawPos < REV_WRAPAROUND_POINT) {
-                rawPos += TURRET_PPR;
+            outputsChanged = false;
+            if (lostEncPos) {
+                manualControl();
+                return;
             }
-            int adjPos = (rawPos + ABS_TICKS_SOUTH + ZERO_OFFSET);
 
-            turret.set(com.ctre.phoenix.motorcontrol.ControlMode.Position, adjPos);
-            outputsChanged = false;
+            if (pos > kFwdWrapAroundPos) {
+                pos -= kTurretPPR;
+            } else if (pos < kRevWrapAroundPos) {
+                pos += kTurretPPR;
+            }
+            int rawPos = (pos - kAbsTicksSouthOffset);
```

Comment:
Should this be plus instead of minus?

```suggestion
            int rawPos = (pos + kAbsTicksSouthOffset);
```